woocommerce-pdf-invoices-packing-slips-print-all-documents.php

```diff
@@ -112,6 +118,17 @@ private function init()
         add_filter('plugin_action_links_' . $this->plugin_basename, [$this, 'add_settings_link']);
     }
 
+    public function add_all_pdf_order_actions($actions, $order)
+    {
+        unset($actions);
+        $actions['all'] = [
+            'url'  => wp_nonce_url(admin_url("admin-ajax.php?action=generate_wpo_wcpdf_all&order_ids=$order->id"), 'generate_wpo_wcpdf_all'),
+            'alt'   => 'Print All PDF Documents',
```

Comment:
Same here, i think that we should create a setting form to let the user choose its language ?